README.md

```diff
@@ -1,2 +1,58 @@
 # cql-exec-fhir-mongo
-A FHIR data provider for the JavaScript CQL Execution project
+
+This project establishes a FHIR-based data source module for use with the CQL Execution Engine. Currently, FHIR 1.0.2 (DSTU2), FHIR 3.0.0 (STU3), FHIR 4.0.0 ,and FHIR 4.0.1 (R4) are supported.
+
+- [Installation](#installation)
+
+  - [Prerequisites](#prerequisites)
+  - [Local Installation](#local-installation)
+  - [Testing](#testing)
+
+- [Usage](#usage)
+
+- [License](#license)
+
+## Installation
+
+### Prerequisites
+
+- [Node.js >=11.15.0](https://nodejs.org/en/)
+- [Git](https://git-scm.com/)
+
+### Local Installation
+
+Clone the source code:
+
+```bash
+git clone https://github.com/projecttacoma/cql-exec-fhir-mongo.git
+```
+
+Install dependencies:
+
+```bash
+npm install
+```
+
+### Testing
+
+Unit tests can be running using the following `npm` command:
+
+```bash
+npm run test
+```
+
+## Usage
+
+The FHIR Data Source expects each patient to be represented as a single FHIR Bundle containing all of the patient's relevant data. The FHIR Data Source queries Mongo to get FHIR data for execution.
```

Comment:
> The FHIR Data Source expects each patient to be represented as a single FHIR Bundle containing all of the patient's relevant data

Technically not true for this repo. We just expect the patient and their data to exist in some mongo database that is provided to the patient source. Can you change the wording?